apps/dotcom/client/src/components/StoreErrorScreen.tsx

```diff
@@ -47,6 +47,11 @@ export function StoreErrorScreen({ error }: { error: Error }) {
 				message = 'The room you are trying to connect to does not exist.'
 				break
 			}
+			case TLIncompatibilityReason.NotAuthorized: {
```

Comment:
We should prbly also have a Forbidden (403), might as well think ahead to that case if you like.

Reply:
this is actually for what '403 forbidden' means in HTTP (i.e. not authorized). it's confusing because http calls 401 'unauthorized' but really it means 'not authenticated'. 

still yeah, we might need to distinguish at some point. let's take care of it then.

Reply:
forget this, i read your comments out of order